src/User.js

```diff
@@ -1,191 +1,46 @@
  class User {
-  constructor(userData) {
-    //may also want to pass in today's date into the constructor
-    this.id = userData.id;
-    this.name = userData.name;
-    this.address = userData.address;
-    this.email = userData.email;
-    this.strideLength = userData.strideLength;
-    this.dailyStepGoal = userData.dailyStepGoal;
-    this.friends = userData.friends;
+   constructor(userData) {
+     this.id = userData.id;
+     this.name = userData.name;
+     this.address = userData.address;
+     this.email = userData.email;
+     this.strideLength = userData.strideLength;
+     this.dailyStepGoal = userData.dailyStepGoal;
+     this.friends = userData.friends;
+     this.hydrationInfo = new HydrationRepository(date);
+     this.activityInfo = new ActivityRepository(date);
+     this.sleepInfo = new SleepRepository(date);
+   }
 
-    //this.hydrationInfo = new HydrationRepository
-        // this.ouncesAverage = 0;
-        // this.ouncesRecord = [];
+   getFirstName() {
+     var names = this.name.split(' ');
+     return names[0].toUpperCase();
```

Comment:
I think moving these methods around will ultimately serve us well in terms of keeping our code DRY, encapsulated, and semantic.